VMAccess/vmaccess.py

```diff
@@ -382,10 +382,12 @@ def _allow_password_auth():
     _set_sshd_config(config, "PasswordAuthentication", "yes")
     ext_utils.replace_file_with_contents_atomic(SshdConfigPath, "\n".join(config))
 
-    if isinstance(MyDistro, dist_utils.UbuntuDistro): #handle ubuntu 22.04 (sshd_config.d directory)
+    if isinstance(MyDistro, dist_utils.UbuntuDistro) or \ #handle ubuntu 22.04 (sshd_config.d directory)
```

Comment:
I think it would be best if we just removed the distro gates so we don't have to add them all over time.

If 50-cloud-init.conf exists, I cannot think of a case where we don't want to update it.  That is, unless vmaccess switches to using its own high-priority config file (e.g. 10-azure-vmaccess.conf) for distros using multi-part configs.

Reply:
Fixed this.